data/application.css

```diff
@@ -76,24 +22,6 @@
     color: @text_color;
 }
 
-scale.temperature trough {
```

Comment:
Can we leave these in? This is actually a different style than what's in the stylesheet right now

Reply:
Sure, I thought the difference was because we did Photos first and figured we'd want to rely on the system stylesheet here. But I'll add it back.